contentmanagement/proposal.md

```diff
@@ -0,0 +1,59 @@
+# Proposal for Incubation
+
+## Subproject name
+
+Jupyter Content Management Extensions (jupyter-incubator/contentmanagement)
+
+## Development team and Advocate
+
+Subproject development team:
+
+* Justin Tyberg, IBM (`@jtyberg `)
+* Dan Gisolfi, IBM (`@vinomaster`)
+* Peter Parente, IBM (`@parente`)
+
+Steering Council Advocate:
+
+* Brian Granger (`@ellisonbg`)
+
+## Subproject goals, scope and functionality
+
+### Concept 
+The Jupyter Notebook allows for the publishing, editing, organizing and deleting of content from a central web interface. Depending on an individual's workflow and collaboration requirements when working on notebook related projects, the collection of content management features they desire may differ from other users.
+
+This incubation proposal pertains to the creation of a package of self-installable content management features that extend the baseline Jupyter Notebook. 
+
+### Goals
+* Establish a collection of content management extensions that can be installed to customize a user instance of a Jupyter Notebook.
+* Allow 1..n content management extensions to be combined to customize an instance of a Jupyter Notebook.  
```

Comment:
I also view one of the goals of this to identify which extensions should be fully integrated into the notebook as default functionality.